### PR TITLE
Create str.md

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/python/concepts/built-in-functions/terms/str/str.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+```py
+str()
+```
+
+## Example 1
+
+Use `str()` to change the datatype of a `float` value `314.0` into a string:
+
+```codebyte/python
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/str/str.md:29]: Let's only have one (1) codebyte snippet per entry (preferably towards the bottom).

File: content/python/concepts/built-in-functions/terms/str/str.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Syntax
+
+```py
+str()
+```
+
+## Example 1
+
+Use `str()` to change the datatype of a `float` value `314.0` into a string:
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/str/str.md:27]:
```suggestion
The `str()` function can change the datatype of a `float` value, `314.0`, into a string:
```

For entries, let's use a voice that is more manual-like ("This is ...") as opposed to instruction/tutorial-like ("Do this ...").

File: content/python/concepts/built-in-functions/terms/str/str.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+```codebyte/python
+print(str(314.0))
+```
+
+## Example 2
+
+Use `str()` to change the datatype of an `int` value `314` into a string:
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/str/str.md:35]:
```suggestion
`str()` can also wrap around other type conversions, like in the example below:
```

File: content/python/concepts/built-in-functions/terms/str/str.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+```codebyte/python
+print(int(314))
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/str/str.md:38]:
```suggestion
print(int(3.14))
```

File: content/python/concepts/built-in-functions/terms/str/str.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+Takes in a value that can be converted into a string, and returns a copy of the value in the `string` datatype.
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/str/str.md:17]:
```suggestion
Takes in a value that can be converted into a string, and returns a copy of the value in the `string` datatype. If nothing is passed into `str()`, then an empty string is returned.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski All finished with my review! 😃 Tag me when you've finished looking over my comments/suggestions.

**Comment by SSwiniarski** [comment]
@Dusch4593, I've updated the PR with your suggestions.

**Review by Dusch4593 — APPROVED** [review]
LGTM! Onward to review 2️⃣

File: content/python/concepts/built-in-functions/terms/str/str.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+Takes in a value that can be converted into a string, and returns a copy of the value in the `string` datatype. If nothing is passed into `str()`, then an empty string is returned.
+
+## Syntax
+
+```py
+str()
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/str/str.md:22]:
```suggestion
str(object)
```

File: content/python/concepts/built-in-functions/terms/str/str.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+## Example 1
+
+The `str()` function can change the datatype of a `float` value, `314.0`, into a string:
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/str/str.md:27]:
```suggestion
The `str()` function changes the datatype of a `float` value, `314.0`, into a string:
```

File: content/python/concepts/built-in-functions/terms/str/str.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+```python
+print(str(314.0))
+```
+
+## Example 2
+
+The `str()` function can also wrap around other type conversions. How could that be done in the example below?
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/str/str.md:35]:
```suggestion
The `str()` function can also wrap around other type conversions.
```

Not sure if these examples are supposed to play out like problems for users to solve, or just demonstrations as we would expect from the documentation. I prefer the latter.

File: content/python/concepts/built-in-functions/terms/str/str.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+```codebyte/python
+print(int(3.14))
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/str/str.md:38]:
```suggestion
print(str(int(3.14)))
```

**Review by yangc95 — COMMENTED** [review]
@SSwiniarski review 2️⃣ complete, some comments and suggestions

**Comment by SSwiniarski** [comment]
@yangc95, I applied your changes.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@SSwiniarski Woohoo, it's live here:

https://www.codecademy.com/resources/docs/python/built-in-functions/str

cc: @Dusch4593

**Pull request merged**